openstudiocore/src/model/FluidCoolerSingleSpeed.cpp

```diff
@@ -0,0 +1,498 @@
+}
+
+bool FluidCoolerSingleSpeed::setWaterOutletNode(const Connection& connection) {
+  return getImpl<detail::FluidCoolerSingleSpeed_Impl>()->setWaterOutletNode(connection);
+}
+
+void FluidCoolerSingleSpeed::resetWaterOutletNode() {
+  getImpl<detail::FluidCoolerSingleSpeed_Impl>()->resetWaterOutletNode();
+}
+
+bool FluidCoolerSingleSpeed::setPerformanceInputMethod(std::string performanceInputMethod) {
+  return getImpl<detail::FluidCoolerSingleSpeed_Impl>()->setPerformanceInputMethod(performanceInputMethod);
+}
+
+bool FluidCoolerSingleSpeed::setDesignAirFlowRateUfactorTimesAreaValue(double designAirFlowRateUfactorTimesAreaValue) {
+  return getImpl<detail::FluidCoolerSingleSpeed_Impl>()->setDesignAirFlowRateUfactorTimesAreaValue(designAirFlowRateUfactorTimesAreaValue);
+}
+
+void FluidCoolerSingleSpeed::autosizeDesignAirFlowRateUfactorTimesAreaValue() {
+  getImpl<detail::FluidCoolerSingleSpeed_Impl>()->autosizeDesignAirFlowRateUfactorTimesAreaValue();
+}
+
+bool FluidCoolerSingleSpeed::setNominalCapacity(double nominalCapacity) {
+  return getImpl<detail::FluidCoolerSingleSpeed_Impl>()->setNominalCapacity(nominalCapacity);
+}
+
+bool FluidCoolerSingleSpeed::setDesignEnteringWaterTemperature(double designEnteringWaterTemperature) {
+  return getImpl<detail::FluidCoolerSingleSpeed_Impl>()->setDesignEnteringWaterTemperature(designEnteringWaterTemperature);
+}
+
+bool FluidCoolerSingleSpeed::setDesignEnteringAirTemperature(double designEnteringAirTemperature) {
+  return getImpl<detail::FluidCoolerSingleSpeed_Impl>()->setDesignEnteringAirTemperature(designEnteringAirTemperature);
+}
+
+bool FluidCoolerSingleSpeed::setDesignEnteringAirWetbulbTemperature(double designEnteringAirWetbulbTemperature) {
+  return getImpl<detail::FluidCoolerSingleSpeed_Impl>()->setDesignEnteringAirWetbulbTemperature(designEnteringAirWetbulbTemperature);
+}
+
+bool FluidCoolerSingleSpeed::setDesignWaterFlowRate(double designWaterFlowRate) {
+  return getImpl<detail::FluidCoolerSingleSpeed_Impl>()->setDesignWaterFlowRate(designWaterFlowRate);
+}
+
+void FluidCoolerSingleSpeed::autosizeDesignWaterFlowRate() {
+  getImpl<detail::FluidCoolerSingleSpeed_Impl>()->autosizeDesignWaterFlowRate();
+}
+
+bool FluidCoolerSingleSpeed::setDesignAirFlowRate(double designAirFlowRate) {
+  return getImpl<detail::FluidCoolerSingleSpeed_Impl>()->setDesignAirFlowRate(designAirFlowRate);
+}
+
+void FluidCoolerSingleSpeed::autosizeDesignAirFlowRate() {
+  getImpl<detail::FluidCoolerSingleSpeed_Impl>()->autosizeDesignAirFlowRate();
+}
+
+bool FluidCoolerSingleSpeed::setDesignAirFlowRateFanPower(double designAirFlowRateFanPower) {
+  return getImpl<detail::FluidCoolerSingleSpeed_Impl>()->setDesignAirFlowRateFanPower(designAirFlowRateFanPower);
+}
+
+void FluidCoolerSingleSpeed::autosizeDesignAirFlowRateFanPower() {
+  getImpl<detail::FluidCoolerSingleSpeed_Impl>()->autosizeDesignAirFlowRateFanPower();
+}
+
+bool FluidCoolerSingleSpeed::setOutdoorAirInletNode(const Node& node) {
```

Comment:
This getter/setter is one that can stay.  It is just a reference as opposed to being connected into the hvac topology through Connection.